Models/Mercurycontext.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using System.ComponentModel.DataAnnotations.Schema;
+using Mercury.Models;
+namespace Mercury
+{
+
+    public class MercuryContext : DbContext
+    {
+        public DbSet<Product> Product { get; set; }
+        public DbSet<User> User { get; set; }
+        public DbSet<Order> Order { get; set; }
+        // public MercuryContext(): base(){
+        //     Database.SetInitializer(new SchoolDBInitializer());
+        // }
+        //public DbSet<Student> Students { get; set; }
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+
+                optionsBuilder.UseMySql("server=localhost; database=MercuryMercury; user=root; pwd=123123;");
```

Comment:
move this to configuration https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-3.1